packages/drivers/routerlicious-driver/src/test/documentService.spec.ts

```diff
@@ -142,4 +162,42 @@ describe("DocumentService", () => {
 		assert.equal(documentService.policies?.summarizeProtocolTree, true);
 		stubbedDeltaConnectionCreate.restore();
 	});
+
+	it("DocumentDeltaStorageService sends fetchReason along with fetchMessages", async () => {
+		// Create fake restWrapperBase
+		const restWrapperBase = new MockRestWrapper();
+		stub(restWrapperBase, "get").callsFake(async (url, headers) => {
+			const headerElements = headers
+				? Object.entries(headers)
+						.map(([key, value]) => `${key}=${value}`)
+						.join("&")
+				: "";
+			const modifiedUrl = `${url}/${headerElements}`;
+			/* Usually the response would be the ops from the server but here we are sending
+			the modified url to check what information is being sent with the link. */
+			const response = {
+				content: modifiedUrl,
+				propsToLog: {},
+				requestUrl: `${modifiedUrl}${"/requestUrl"}`,
+			};
+			return response;
+		});
+		const testLogger = new MockLogger();
+		const documentDeltaStorageService = new DeltaStorageService(
+			"https://mock.url/deltaStorageUrl",
+			restWrapperBase,
+			testLogger.toTelemetryLogger(),
+		);
+		const message = await documentDeltaStorageService.get(
+			"tenantId",
+			"id",
+			1,
```

Comment:
This is a 1 and the url has a 0, am I missing something?

Reply:
The "get" method subtracts 1 off the "from" param because of inclusive vs exclusive range

Reply:
Ok sounds good, feel free to resolve

Reply:
Maybe worth commenting in this test